metals/src/main/scala/scala/meta/internal/metals/BloopServers.scala

```diff
@@ -388,7 +387,6 @@ final class BloopServers(
         bloopCreatedByMetalsFilePath,
         maybeBloopJvmProperties,
         maybeJavaHome,
-        bloopJsonUpdateCause,
```

Comment:
I decided that since we are changing things in the settings it just adds additional work to user to then click to restart Bloop server. Especially, that a lot of users might not understand it fully.